### ISSUE TITLE
[BUG] Websocket Max Frame Size is 16Kb

### ISSUE THREAD

**Issue by NghiaTranUIT** [ProxymanApp/Proxyman#1569]
## Description
For some reason, the Websocket message is trimmed unexpectedly. We've checked that we set the MaxFrameSize is UInt32.max, but the WS Message is cut at 16Kb.

![Screenshot 2023-03-06 at 15 50 23](https://user-images.githubusercontent.com/5878421/223061788-995fbc3a-21dd-4195-9ba7-75e92c89c921.jpg)

## Steps to Reproduce
1. Make a huge WS Message
2. The WS Message is missing

## Current Behavior
The WS Message is trimmed

## Expected Behavior
- Should show entire WS message

**Comment by invisible-debug** [comment]
This is still critical. When is the fix planned?

**Comment by invisible-debug** [comment]
@NghiaTranUIT This is still critical. When is the fix planned?

**Comment by NghiaTranUIT** [comment]
@invisible-debug I can't reproduce this bug. I tried to setup a NodeJS WS Server, and I can sent 5Mb JSON file without any problems.

Can you share with me what Backend library you're using? Python, Ruby and what the network lib is?

**Comment by invisible-debug** [comment]
@NghiaTranUIT Try to send not just JSON, but a MESSAGE frame packet frame with a content length of more than 19000 characters.
Here is a link to the stomp protocol:
https://stomp.github.io/stomp-specification-1.2.html#MESSAGE

**Comment by invisible-debug** [comment]
<img width="617" alt="cutted_message" src="https://github.com/ProxymanApp/Proxyman/assets/174007516/12d14f7a-db48-40e9-ab7a-c6df098e5a8d">

**Comment by NghiaTranUIT** [comment]
@invisible-debug I guess I need to set up a sample Stomp Websocket Server to test it.

Since now, I just tested with a normal Websocket from NodeJS server and it works fine.

**Comment by NghiaTranUIT** [comment]
@invisible-debug any chance to export your Websocket and share to us at support@proxyman.io ?

**Comment by invisible-debug** [comment]
@NghiaTranUIT 
Unfortunately, I cannot disclose the corporate json. But I tried to copy it from Charles, and then send it through the local mock server and Map Remote. And the bug is not reproducible :(
But there is one point that distinguishes a bug situation from a bug-free situation. Here is another screenshot where I showed it. This frame appears when the cursor hovers over the json body. Is this the reason why the json is not fully displayed?
<img width="661" alt="frame_cutted_json" src="https://github.com/ProxymanApp/Proxyman/assets/174007516/fd04b0a7-9aed-45b6-b6ff-4ee2f5cac16c">

**Comment by NghiaTranUIT** [comment]
@invisible-debug  thanks, it could be. I will try to disable it.

May I ask:
- Do your client receive a full JSON websocket message? 

If yes, it's definitely a bug when displaying a JSON, not a bug when sending/receiving the WS message.

**Comment by invisible-debug** [comment]
@NghiaTranUIT Yes, it is transmitted correctly. The problem is only in the display

**Comment by NghiaTranUIT** [comment]
@invisible-debug here is the Beta build: https://download.proxyman.io/beta/Proxyman_5.5.0_Fix_monaco_editor_renders_a_long_line.dmg

I increase the limit to 50,000 (default is 10,000).

**Comment by invisible-debug** [comment]
@NghiaTranUIT Unfortunately, this did not solve the problem. And why do I see the brackets in red? Because there are no closing brackets in the editor?
Maybe make a build with a complete lack of rendering? To exclude that he affects the body
<img width="611" alt="frame_cutted_json_2" src="https://github.com/ProxymanApp/Proxyman/assets/174007516/85be6eb7-9bc9-4e8c-8370-70aa4f8bca38">
